ingester/neo4j_ingester.py

```diff
@@ -64,6 +64,10 @@ def add_mc(self, model_card):
             xai_id = base_mc_id + "-xai"
             self.db.insert_xai_analysis_metadata(base_mc_id, xai_id, xai_analysis)
 
+        foundational_mc_id = model_card['foundational_model']
+        if foundational_mc_id is not None:
```

Comment:
can't we check foundational_mc_id like
if foundational_mc_id? instead of foundational_mc_id is not None?